third_party/terraform/utils/provider.go.erb

```diff
@@ -51,6 +51,14 @@ func Provider() terraform.ResourceProvider {
 				}, nil),
 			},
 
+			"billing_project": &schema.Schema{
```

Comment:
Can you update https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/website/docs/guides/provider_reference.html.markdown with the field as well?